dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -1426,6 +1528,431 @@ async fn cmd_db_snapshot_info(
     Ok(())
 }
 
+/// List all region replacement requests
+async fn cmd_db_region_replacement_list(
+    _opctx: &OpContext,
+    datastore: &DataStore,
+    fetch_opts: &DbFetchOptions,
+    args: &RegionReplacementListArgs,
+) -> Result<(), anyhow::Error> {
+    let ctx = || "listing region replacement requests".to_string();
+    let limit = fetch_opts.fetch_limit;
+
+    let requests: Vec<RegionReplacement> = datastore
+        .pool_connection_for_tests()
+        .await?
+        .transaction_async(|conn| async move {
```

Comment:
out of interest, why is this a transaction? IIUC only a single query is performed...

Reply:
I have no answer, removed those transaction calls in a155af913

Reply:
I have no answer, removed those transaction calls in a155af913